modules/flashrom

```diff
@@ -2,11 +2,11 @@ modules-$(CONFIG_FLASHROM) += flashrom
 
 flashrom_depends := pciutils $(musl_dep)
 
-flashrom_version := 1776bb46ba6ea3d1ab2ec3f0cd88158aabed7400
+flashrom_version := eace095b15eb034e42d97202cad70ce979d8ca38
 flashrom_dir := flashrom-$(flashrom_version)
 flashrom_tar := $(flashrom_dir).tar.gz
 flashrom_url := https://github.com/flashrom/flashrom/archive/$(flashrom_version).tar.gz
-flashrom_hash := 65e262ca4428a0ceddd73f481ed0d8444393b73a78469f266a4457dfc834ecb7
+flashrom_hash := 9eb81ec00d6c0ebe8ccd1b6a242b61e2b6b242ca1a6a66650d2701ca2a1f1972
```

Comment:
The flashrom update seems to cause a problem with the `flashrom_progress` progress bar on Librem 14.  The progress bar always gets stuck at 0% (this happens if the verbose output parsing fails to find a token it's looking for to advance to the next state)

Flashing actually works, but Heads gets stuck showing the progress bar.  (You have to wait a few minutes to gamble that the flash is done, then force off.)

It works on Librem 13v2 though.

TBH, `flashrom_progress` is a huge pain.  It has frozen like this occasionally since I've been involved with Heads.  I made some improvements to reduce it but it still happens (<1% of the time I think, but it's a huge pain for customers not knowing what to do, risking a brick).  The way it tries to parse flashrom's verbose output is fundamentally flawed, it always risks getting stuck if a token is split across buffers and wastes tons of CPU power churning through output.  (On some Librems the actual flash usually finishes well before the progress bar actually finishes, because it takes so much work to do the process bar.)

@tlaurion What would you think about removing the progress bar?  The dream would be to rework it (maybe doing it in flashrom itself, or at least providing a reliably-parseable progress output), but the current implementation is a huge pain, I don't feel it adds enough value to outweigh the pain.  For comparison the read step takes a comparable amount of time and does not have a progress bar.

Otherwise we have to block this PR until the progress bar works again on Librem 14 (and any other boards where it's broken, there are probably others).

Reply:
@JonathonHall-Purism see #1755 which also impacts #1395. Should be fixed upstream, I don't think its Heads problem at this point.

Reply:
Heads in-house progress output removed with [d2d33bc](https://github.com/linuxboot/heads/pull/1753/commits/d2d33bce6e3e104793342647b6c81623c1f10257)